dist-git.spec

```diff
@@ -90,65 +89,58 @@ getent group gen-acls > /dev/null || \
 
 getent passwd gen-acls > /dev/null || \
     useradd -r -g gen-acls -G packager -s /bin/bash \
-            -d %{_sharedstatedir}/dist-git/git gen-acls
+            -d /srv/git gen-acls
 
 
 %install
 # ------------------------------------------------------------------------------
-# /usr/share/ .... static files
+# /usr/local/bin ........... scripts
 # ------------------------------------------------------------------------------
-install -d %{buildroot}%{_datadir}/dist-git
-
-cp -a scripts/dist-git/* %{buildroot}%{_datadir}/dist-git/
+install -d %{buildroot}/usr/local/bin/
+cp -a scripts/dist-git/* %{buildroot}/usr/local/bin/
 
 # ------------------------------------------------------------------------------
 # /etc/ .......... config files
 # ------------------------------------------------------------------------------
 install -d %{buildroot}%{_sysconfdir}/dist-git
+install -d %{buildroot}%{_sysconfdir}/gitolite
 install -d %{buildroot}%{_sysconfdir}/httpd/conf.d/dist-git
-install -d %{buildroot}%{_sysconfdir}/cron.d/dist-git
 mkdir -p   %{buildroot}%{_unitdir}
 
 cp -a configs/dist-git/dist-git.conf  %{buildroot}%{_sysconfdir}/dist-git/
-cp -a configs/gitolite/gitolite.rc    %{buildroot}%{_sysconfdir}/dist-git/
 cp -a configs/httpd/dist-git.conf     %{buildroot}%{_sysconfdir}/httpd/conf.d/
+cp -a configs/gitolite/gitolite.rc    %{buildroot}%{_sysconfdir}/gitolite/
 cp -a configs/httpd/ssl.conf.example  %{buildroot}%{_sysconfdir}/httpd/conf.d/
 cp -a configs/httpd/dist-git/* %{buildroot}%{_sysconfdir}/httpd/conf.d/dist-git/
-cp -a configs/cron/*           %{buildroot}%{_sysconfdir}/cron.d/dist-git/
 cp -a configs/systemd/*        %{buildroot}%{_unitdir}/
 
+install -d %{buildroot}%{_sysconfdir}/gitolite
+install -d %{buildroot}%{_sysconfdir}/gitolite/conf
+install -d %{buildroot}%{_sysconfdir}/gitolite/logs
+install -d %{buildroot}%{_sysconfdir}/gitolite/local
+install -d %{buildroot}%{_sysconfdir}/gitolite/local/VREF
+install -d %{buildroot}%{_sysconfdir}/gitolite
+install -d %{buildroot}%{_sysconfdir}/gitolite/hooks
+install -d %{buildroot}%{_sysconfdir}/gitolite/hooks/common
+
```

Comment:
While we are on major rewrite, would it be possible to somehow ensure that dist-git machine can be configured so it has only one "persistent storage" mount point?  Ideally, we should be able to:
  * install dist-git and copr-dist-git packages
  * mount the persistent volumes
  * start the service

What I'm doing so far is that I need to mount the volume first, and then install the package .... and there's quite some stuff maintained by RPM on the mounted volume (which somewhat complicates migrations).

IOW, it would be nice if there was single-data-point (un)mountable without the need to re-install the package.  But maybe I'm doing something wrong ..